pkg/service/replay/replay.go

```diff
@@ -89,21 +89,30 @@ func (r *Replayer) Start(ctx context.Context) error {
 			utils.LogError(r.logger, err, "failed to stop recording")
 		}
 	}()
+	if _, err := os.Stat(r.config.Path); os.IsNotExist(err) {
+		errMsg := "keploy directory does not exist"
```

Comment:
Error should be "no test-sets found, please use record cmd to record test case" something like this.

Reply:
This is for checking if the keploy directory exists or not. We can change the message in errmsg down below.

Reply:
logging it as keploy directory, there is no use case for keploy directory. Test-set is a useCase where test cases are present, logging it as test set make more meaning.